react/src/components/HazmapperHeader/HazmapperHeader.module.css

```diff
@@ -0,0 +1,9 @@
+.root {
```

Comment:
can this have a fixed height of --hazmapper-top-navbar-height from here https://github.com/TACC-Cloud/hazmapper/blob/main/react/src/hazmapper.css#L16.  

also, there is placeholder in MapProject ([here](https://github.com/TACC-Cloud/hazmapper/blob/main/react/src/pages/MapProject/MapProject.tsx#L154) and the style [here](https://github.com/TACC-Cloud/hazmapper/blob/main/react/src/pages/MapProject/MapProject.tsx#L154)) that can not be removed and this used 💯

Reply:
Related to the above, I think we should unify the names of this component and the CSS variable.  🤔  

what about:

`HeaderNavBar` and `hazmapper-header-navbar-height`